src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
+  const priceForDay = 40;
+  const count = days * priceForDay;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
+
+  return count - discount;
```

Comment:
Use `if` with `return` to simplify the function. There's no need for an `else` because if the condition is met, the function will return early.